packages/site/mosaic.config.js

```diff
@@ -40,7 +44,7 @@ module.exports = deepmerge(siteConfig, {
      * Access from your browser as http://localhost:3000/mosaic
      */
     {
-      disabled: process.env.NODE_ENV !== 'development',
+      // disabled: process.env.NODE_ENV !== 'development',
```

Comment:
oops

Reply:
I have added `NODE_ENV=development` into the `env.local` file so I dont need to do this anymore.  If you rebase you should pick it up...